js/loginbuttonlogic.js

```diff
@@ -1,5 +1,17 @@
-function openLogin() {
-    console.log(document.getElementById("overlay"));
-    document.getElementById("overlay").style.visibility= 'visible';
-    document.getElementById("logInModal").style.visibility= 'visible';
+const logInButton = document.querySelector('.logInButton'),
+    overlay = document.querySelector('.overlay'),
+    logInModal = document.querySelector('.modal'),
+    closeLogInModalButton = document.querySelector('.close-modal');
```

Comment:
i, personally, use `document.querySelector` over `getElementById()` or `getElementsByClassName()`. i prefer `querySelector` because it will accept the name of a class, an id, or a tag, all in one function. note how i'm targeting class names here by passing in strings like `'.logInButton'` (the `.` is meant to mimic the class selector from the world of CSS)

more on [document.querySelector](https://careerkarma.com/blog/javascript-queryselector-vs-getelementbyid/)